### ISSUE TITLE
Tree: aria-posinset is not set for child nodes

### ISSUE THREAD

**Issue by WilliamSaedan** [primefaces/primevue#7905]
### Describe the bug

In the `Tree` and `TreeSelect` components, there exists accessibility issues surrounding the `aria-posinset` value. Currently, the value is not set for children nodes in the `Tree`.

### Pull Request Link

https://github.com/primefaces/primevue/pull/7904

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primevue.org/tree/

### Environment

Latest PrimeVue 4.3.6

### Vue version

3.5.13

### PrimeVue version

4.3.6

### Steps to reproduce the behavior

1. Open `Tree` dropdown such that it displays children
2. Inspect the childs' `aria-posinset` property or run an automated axe accessibility test
3. You should see that `aria-posinset` is incorrectly set to `NaN`

### Expected behavior

The `aria-posinset` property of the `TreeNodes` should be set to a numerical value corresponding to its position in the set.

**Comment by tugcekucukoglu** [comment]
Please track at: https://github.com/primefaces/primevue/issues/7899
